### ISSUE TITLE
Equation solvers not used?

### ISSUE THREAD

**Issue by pannal** [Diaoul/subliminal#715]
I can't find any usage of the functions `solve_movie_equations` and `solve_episode_equations`. I thought I removed them in Sub-Zero's subliminal usage, but it seems like the usage was never there.

Am I missing something?

https://github.com/Diaoul/subliminal/blob/dd74383d1cba82829ce720f2e439a65d13ffe7ef/subliminal/score.py#L151 and https://github.com/Diaoul/subliminal/blob/dd74383d1cba82829ce720f2e439a65d13ffe7ef/subliminal/score.py#L197

**Comment by Diaoul** [comment]
See the note in the docs:
https://github.com/Diaoul/subliminal/blob/master/subliminal/score.py#L6-L9

**Comment by Diaoul** [comment]
I solve the equations during development and use the result to set the score dicts.
> To avoid unnecessary dependency on [sympy](http://www.sympy.org/) and boost subliminal's import time, the resulting scores are hardcoded here and manually updated when the set of equations change.

**Comment by pannal** [comment]
Yeah I've seen that. Thank you, I was just wondering whether I got that wrong or not as [someone is claiming that with Subliminal.bundle (Sub-Zero's predecessor) the scores were much lower](https://github.com/pannal/Sub-Zero.bundle/issues/197). So I've checked the old < 1.0.0 subliminal versions against the 1.1.1 and couldn't find a difference in generic scoring.

I guess he must be mixing movie scores with series scores or something like that.
